packages/core/src/services/db/backends/ephem-db.provider.ts

```diff
@@ -88,9 +93,9 @@ export class EphemDBProvider implements BaseDBProvider {
    * Should be more efficient than simply calling `set` repeatedly.
    * @param objects A series of objects to put into the database.
    */
-  public async bulkPut(objects: DBObject[]): Promise<void> {
+  public async batch(objects: DBObject[]): Promise<void> {
```

Comment:
I believe this should be `public async batch(ops: DBOperation[]): Promise<void> {`

Reply:
Whoops totally right -- updating